tests/e2e/specs/userprofile.js

```diff
@@ -9,4 +16,63 @@ describe('User Profile', () => {
       .should('be.visible')
       .should('be.disabled')
   })
+  it('Increases the font size', () => {
+    resetFontSize()
+    cy.visit('/#/user-profile')
+    expect(parseFloat(INITIAL_FONT_SIZE)).to.be.equal(getCurrentFontSize())
+    const clicks = 3
+    // TODO: https://github.com/cypress-io/cypress/issues/695#issuecomment-379817133
+    // Also had to use Promises in order to locate right element with Cypress
```

Comment:
Note to self: double check

Reply:
The promise approach actually fixed it, so we don't need it any more. But good to know about this hack, as it could be useful later (p.s. comparing with Selenium, Cypress needs very little hacking)